recipes/fast-ref

```diff
@@ -0,0 +1,3 @@
+(fast-ref :repo "timotaysci/fast-ref"
+        :fetcher github
+        :files (:defaults "README.org"))
```

Comment:
Typically MELPA avoids distributing the READMEs unless there's a good reason (let me know if there is, and I missed it).

Reply:
File now compiles clean - sorry for the oversight. 
readme removed